README.md

```diff
@@ -3,40 +3,70 @@
 > Official GitHub Action for [OSSF scorecard](https://github.com/ossf/scorecard).
 
 ## Installation
-The simplest and quickest way to install Scorecards's GitHub action is from the [GitHub's marketplace](https://github.com/marketplace/actions/ossf-scorecard-action).
-**WARNING**: the action does *not* support private repositories.
+**NOTE**: The Scorecards GitHub Action does not support private repositories. Private repositories can run Scorecards manually by following the [standard installation instructions)[https://github.com/ossf/scorecard#installation].
```

Comment:
fyi, we do support private repositories now. `publish_results` is set to false and the value provide by developers in the workflow is ignored. Let's update the example workflow and the doc to reflect it?

Reply:
fyi, this is the text I'm using for the default workflow https://github.com/actions/starter-workflows/pull/1345/files

Reply:
Done.